src/lib/list/list.scss

```diff
@@ -116,29 +120,23 @@ $mat-list-item-inset-divider-offset: 72px;
     font-size: $icon-size;
     box-sizing: content-box;
     border-radius: 50%;
-    padding: 4px;
+    padding: $mat-list-icon-padding;
+
+    ~ .mat-divider-inset {
+      @include mat-inset-divider-offset($icon-size + (2 * $mat-list-icon-padding));
+    }
   }
 
   .mat-divider {
-    position: absolute;
+    position: absolute !important; // this is done to override card inset styles
```

Comment:
Rather than using `!important`, perhaps the selector can be made more specific or the card one can be less specific?

Reply:
The original nesting was the way to make it more specific. I don't think that modifying the card styles will be as elegant. I'll add back the nesting for now unless you have an alternative suggestion